_data/tools.json

```diff
@@ -962,6 +962,16 @@
       "note": null,
       "type": "SAST"
    },
+   {
+      "title": "DeepSource",
+      "url": "https://deepsource.io/",
+      "owner": null,
+      "license": "Commercial",
+      "platforms": "SaaS or On-Premises",
+      "note": "DeepSource helps you automatically find and fix issues in your code during code reviews, such as bug risks, anti-patterns, performance issues, and security flaws. It takes less than 5 minutes to set up with your Bitbucket, GitHub, or GitLab account. It works for Python, Go, Ruby, and JavaScript.
+",
```

Comment:
Please move the closing quote and comma to the previous line to address the JSON validity issue.

Reply:
Sure, thanks for the suggestion.